pkg/database/realm.go

```diff
@@ -1323,6 +1323,18 @@ func (db *Database) MaximumUserReportTimeout() (time.Duration, error) {
 	return timeout, nil
 }
 
+func (db *Database) FindE2ETestRealms() ([]*Realm, error) {
```

Comment:
Nit: Should be `ListE2ETestRealms()`. Also, there can only be one e2e test realm (database constraint), and I thought we already had a function to find it.

Reply:
So maybe change to `*Realm` instead

Reply:
I missed the DB constraint - I could not find an existing function for this